cmd/release/config/config.go

```diff
@@ -74,3 +91,89 @@ func read(r io.Reader) (*Config, error) {
 	}
 	return &c, nil
 }
+
+func (c *Config) String() (string, error) {
```

Comment:
I didn't implement this because of the auth struct and din't want to push all of that out to screen and "leak" keys. I think we should probably output the config in a more cli type of way, like a tablewriter. If we want the JSON, we can cat the file.